tests/test_Voting.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
```

Comment:
Way more tests needed, each method should have at least one test (preferably more i.e. boundary, valid, invalid etc.), only ones that shouldn't are those that handle ctx/ payload (should only be the commands, otherwise move handling of those to the command/ on_reaction methods)